krkn/scenario_plugins/network_chaos_ng/modules/templates/pod_module.j2

```diff
@@ -3,10 +3,24 @@ kind: Pod
 metadata:
   name: modtools
```

Comment:
lets add a regex here to the end of the name so that it isn't conflicting with previous runs, we're hitting this currently with a customer

Reply:
done!